in_toto/common_args.py

```diff
@@ -196,6 +196,13 @@ def parse_password_and_prompt_args(args):
     "help": ("generate metadata using dsse (experimental)."),
 }
 
+STATEMENT_ARGS = ["--use-statement"]
```

Comment:
I'm worried `statement` is ambiguous, could you update this to be `attestation` instead?

Reply:
Ideally, `--generate-attestation`.